### PR TITLE
Use GitHub instead of Travis

### PR DESCRIPTION
**Pull request by liZe**
https://phabricator.endlessm.com/T31180

Secrets have to be added to this repository to allow deployments on Metabase (see #129).

Deployments on ReadTheDocs are possible using a simple webhook, no CI required.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get Docker Image
+        uses: actions/download-artifact@v2
+        with:
+          name: image
+      - name: Load Docker Image
+        run: docker load < docker-image.tar
+      - name: Install Docker Images
```
Comment (adarnimrod) [inline .github/workflows/tests.yml:35]: I think this step could be replaced with [services](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices).

File: .github/workflows/metabase.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: Azafea Metabase Documentation
+on:
+  push:
+    branches:
+      - master
+
+jobs:
```
Comment (adarnimrod) [inline .github/workflows/metabase.yml:7]: Aren't we missing the update to RTD?

**Review by adarnimrod — CHANGES_REQUESTED** [review]
I had 2 comments. The one about sing services isn't critical, but I think that migrating the RTD update is.

File: .github/workflows/metabase.yml (hunk 2 above)
Comment (wjt) [reply .github/workflows/metabase.yml:7]: We discussed this verbally – RTD is updated by a web hook.

**Pull request merged**